datanode/sqlstore/migrations/0010_live_liquidity_provisions.sql

```diff
@@ -0,0 +1,57 @@
+-- +goose Up
+
+create table live_liquidity_provisions
+(
+    id BYTEA NOT NULL,
+    party_id BYTEA,
+    created_at TIMESTAMP WITH TIME ZONE NOT NULL,
+    updated_at TIMESTAMP WITH TIME ZONE NOT NULL,
+    market_id BYTEA,
+    commitment_amount HUGEINT,
+    fee NUMERIC(1000, 16),
+    sells jsonb,
+    buys jsonb,
+    version BIGINT,
+    status liquidity_provision_status NOT NULL,
+    reference TEXT,
+    tx_hash BYTEA NOT NULL,
+    vega_time TIMESTAMP WITH TIME ZONE NOT NULL,
+    PRIMARY KEY (id)
+);
+
+-- +goose StatementBegin
+CREATE OR REPLACE FUNCTION update_live_liquidity_provisions()
+   RETURNS TRIGGER
+   LANGUAGE PLPGSQL AS
+$$
+BEGIN
+
+DELETE FROM live_liquidity_provisions
+WHERE id = NEW.id;
+
+-- We take into consideration Liquidity provisions with statuses:
+-- Active (1), Undeployed (5), Pending (6)
+IF NEW.status IN('STATUS_ACTIVE', 'STATUS_UNDEPLOYED', 'STATUS_PENDING')
+    THEN
+        INSERT INTO live_liquidity_provisions(id, party_id, created_at, updated_at,
+                    market_id, commitment_amount, fee, sells, buys, version, status, reference, tx_hash, vega_time)
+        VALUES(NEW.id, NEW.party_id, NEW.created_at, NEW.updated_at,
+                NEW.market_id, NEW.commitment_amount, NEW.fee, NEW.sells,
+                NEW.buys, NEW.version, NEW.status, NEW.reference, NEW.tx_hash, NEW.vega_time);
+END IF;
+
+RETURN NEW;
+END;
+$$;
+-- +goose StatementEnd
+
+select create_hypertable('current_liquidity_provisions', 'vega_time', chunk_time_interval => INTERVAL '1 day');
```

Comment:
convention in datanode is that any table that is 'current'  does not contain contain historical data, i.e. is not a hypertable.  Ideally this table would be called just 'liquidity_provisions', could could we add an altertable command to the migration to rename the table?